tests/test_tuplists.py

```diff
@@ -192,10 +192,21 @@ def fmt(_tup):
         self.assertEqual(a, self.prop1)
 
```

Comment:
missing test: when a list of length 1 is passed.

Reply:
exactly and it currently doesn't work in that case

Reply:
It should work for both cases now.
I had to change the logic quite a bit from my previous commit to make it works for is_list=True